src/components/Integrations/BigQuery/ControlPanel/index.ts

```diff
@@ -0,0 +1,2 @@
+import ControlPanel from './ControlPanel';
```

Comment:
How about avoiding the use of typescript components as we did last time? that way there is only one file for each component.

Reply:
I'm agree. In this project we don't usually use typescript in components. I think that it would be better to keep it in this way.